rts/lra/pom.xml

```diff
@@ -170,6 +177,18 @@
           </properties>
       </profile>
 
+        <profile>
+            <id>trace.thorntail.test</id>
+            <activation>
+                <property>
+                    <name>trace.thorntail.test</name>
+                </property>
+            </activation>
+            <properties>
+                <thorntail.test.logging>-Dthorntail.logging=TRACE</thorntail.test.logging>
```

Comment:
We should only log narayana specific messages which can be done by appending the log category to the property name. So this line should be
```
<thorntail.test.logging>-Dthorntail.logging.com.arjuna=TRACE -Dthorntail.logging.io.narayana.lra=TRACE</thorntail.test.logging>
```

There is a similar line in TCK pom:
```
<lra.tck.suite.thorntail.trace.params>-Dthorntail.logging.com.arjuna=TRACE -Dthorntail.logging.io.narayana.lra=TRACE</lra.tck.suite.thorntail.trace.params>
```

BTW why do we need to repeat this property with a different name. Can they be combined.

Reply:
Okay, I will change the property to "-Dthorntail.logging.com.arjuna=TRACE"

In TCK there is same property repeated because maybe in case, we want to enable, only TCK trace logs. But for now it is not the case, so I will remove the duplication part and will apply same property to both places, i.e. "thorntail.test.logging"